scripts/verify-local-dependencies.js

```diff
@@ -77,6 +77,7 @@ const roleRules = [
       '@backstage/core-compat-api',
       '@backstage/dev-utils',
       '@backstage/frontend-defaults',
+      '@backstage/frontend-dynamic-feature-loader',
```

Comment:
Hmm, is this really needed? can be cleaned up in a followup though

Reply:
Oh, sure. I also guess we can remove it.
I'll create a PR for it.